src/lib/glifApi.ts

```diff
@@ -0,0 +1,96 @@
+import { type ApiAllowanceResponse } from '@/type'
+import { config } from '@/config'
+
+/**
+ * Get the allowance for a verfier from the API.
+ *
+ * @param {string} address - The address to get the allowance for.
+ * @returns {Promise<ApiAllowanceResponse>} ApiAllowanceResponse - The response from the API.
+ */
+export const getAllowanceForVerifier = async (
```

Comment:
`glifAPi` = Lotus RPC API, which already is somewhat integrated in the app (via the VerifyAPI stuff).